### DISCUSSION TITLE
testing in postman

### DISCUSSION THREAD

**Discussion by Nutchareessw** [Azure-Samples/ms-identity-python-webapp#77] [Q&A]
Hi, I am very new to the azure service. I have followed the step and it's working well when I test it on chrome. So, I am trying to access http://localhost:5000/graphcall in postman but I am not be able to call it. I only can get an access token from postman but when I called http://localhost:5000/graphcall the postman link me back to the login page. Can anyone explain to me and Is anyway to test in the postman. 

Thank you.

**Comment by rayluo** [comment]
This sample is designed to be a web app, not a web api. (See also [What is the difference between a Web application and Web API?](https://stackoverflow.com/questions/37643506/what-is-the-difference-between-a-web-application-and-web-api).)

Browser maintains your signed-in session automatically. Postman would probably require you to tweak some cookie settings. Browser is a better tool for your job here.
